doc/conf.py

```diff
@@ -211,14 +211,14 @@
 linkcheck_ignore = [
     'https://127.0.0.1:8443/1.0',
     'https://web.libera.chat/#lxd',
+    'https://docutils.sourceforge.io/docs/ref/rst/directives.html#list-table',
     'http://localhost:8001',
-    r'/lxd/en/latest/api/.*',
```

Comment:
Why are you removing that?

Reply:
I just checked the code of the linkcheck, and it uses "match" - which (if I understand the docs correctly) only matches at the beginning of the string. So `r'/api/.*` should be the same as  `r'^/api/.*`.
I haven't tested it though, so if you have, I'll shut up. ;)
But then I prefer to still keep both, but add the `^`.

Reply:
My bad, I wrongly assumed that `r'/api/.*'` would supersede the ` r'/lxd/en/latest/api/.*'` one. I've now reverted it but apparently testing was still passing.

Reply:
Builds that use the Makefile (thus local builds and GitHub builds) will always have `/api/`, while Read the docs builds will have `/lxd/en/...`.
Which means we could probably remove it after all, since we don't run linkchecks on Read the docs ... but it's probably cleaner to keep it just in case.

Reply:
I agree that it's cleaner to keep it.